pages/linux/wol.md

```diff
@@ -0,0 +1,28 @@
+# wol
+
+> Client for sending Wake-on-LAN magic packets.
+> More information: <https://sourceforge.net/projects/wake-on-lan/>.
+
+- Send a WoL packet to a device:
+
+`wol {{mac_adress}}`
+
+- Send a WoL packet to a device in another subnet based on its IP:
+
+`wol --ipaddr={{ip_address}} {{mac_adress}}`
+
+- Send a WoL packet to a device in another subnet based on its hostname:
+
+`wol --host={{hostname}} {{mac_adress}}`
+
+- Send a WoL packet to a specific port:
+
+`wol --port={{porn_number}}`
```

Comment:
```suggestion
`wol --port={{port_number}}`
```

Reply:
A Freudian slip while getting the inspiration to do this half asleep.